packages/cmf/src/reducers/collectionsReducers.js

```diff
@@ -54,6 +105,20 @@ function deleteCollectionElement(state, action) {
  */
 function updateCollectionElement(state, action) {
 	if (action.operations.update) {
+		const collection = state.get(action.id);
+		if (Map.isMap(collection)) {
+			return updateMapElements(state, action);
+		}
+		else if (List.isList(collection)) {
+			return updateListElements(state, action);
+		}
+		else {
+			throw new Error('CMF collection update is only compatible with ImmutableJs List and Map');
+		}
+	}
+	return state;
```

Comment:
sorry I thought you were talking about the return `above`.
Cleaned this dead code, thanks